api-reference/changes/apply-change.mdx

```diff
@@ -1,3 +1,61 @@
 ---
 openapi: post /v1/changes/{changeId}/apply
----
\ No newline at end of file
+---
+
+<RequestExample>
+```javascript Node.js
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<NOVU_API_KEY>');
+await novu.changes.applyOne("changeId");
+```
+
+```php PHP
+use Novu\SDK\Novu;
+
+$novu = new Novu(<NOVU_API_KEY>);
+$novu->applyChange($changeId, []);
+```
+
+```ruby Ruby
+require 'novu'
+
+client = Novu::Client.new('<NOVU_API_KEY>')
+
+client.apply_change(`<change_id≥`)
+```
+
+```python Python
+from novu.api import ChangeApi
+
+url = "https://api.novu.co"
+api_key = "<NOVU_API_KEY>"
+novu = ChangeApi(url, api_key).apply(
+    change_id = "<A change ID to apply>",
+)
+```
+
+```java Java
+import co.novu.sdk.Novu;
+
+public class Main {
+    public static void main(String[] args) {
+        String apiKey = "<NOVU_API_KEY>";
+
+        Novu novu = new Novu(apiKey);
+        Novu.applyChange(changeId);
```

Comment:
`changeId` needs to be a string in quote
 here else it feels like you are using a variable that is not declared.